defaults.h

```diff
@@ -222,6 +222,12 @@ struct nd_opt_dnssl_info_local {
 #endif
 #endif
 
+/* Captive Portal RFC 8910 */
+
+#ifndef ND_OPT_CAPTIVE_PORTAL
+#define ND_OPT_CAPTIVE_PORTAL 37
```

Comment:
FYI/FWIW
That `37`  matches the `37` in section 2.3 "The Captive-Portal IPv6 RA Option"
of https://www.rfc-editor.org/rfc/rfc8910.html